### PR TITLE
Updated to latest dev branch

### PR DESCRIPTION
**Pull request by CepiPerez**

### CODE REVIEW

File: src/client/notifier.py [hunk 1]
```diff
@@ -26,31 +26,44 @@
 from QtMobility.MultimediaKit import QMediaPlayer
 from PySide.phonon import Phonon
 from wadebug import NotifierDebug
+import dbus
 
 class Notifier():
-	def __init__(self,audio=True,vibra=False):
+	def __init__(self,audio=True,vibra=True):
```
Comment (tgalal) [inline src/client/notifier.py:32]: Never change vibra to True during development. QML errors don't show if vibra is on

Comment (CepiPerez) [inline src/client/notifier.py:32]: Sure? I can see them.
I thought it was because of console.log(), and now we have consoleDebug()

Comment (tgalal) [inline src/client/notifier.py:32]: I'm not talking about our debugging output in qml. I 'm talking about normal qml  thrown errors. You'll never see them if vibra is on. But don't resubmit the pull request for this, I'll update it this time.

**Comment by tgalal** [comment]
file:///opt/waxmppplugin/bin/wazapp/UI/Settings/SetBackground.qml:43: ReferenceError: Can't find variable: recorded

Now this is something to fix and resubmit :P

btw, that background changing feature is fucking cool!

**Pull request closed without merging**